.ci/install_proxy.sh

```diff
@@ -23,10 +23,23 @@ source "${cidir}/lib.sh"
 
 clone_build_and_install "github.com/clearcontainers/proxy"
 start_proxy_cmd="sudo systemctl start cc-proxy"
+upstart_source_file="${cidir}/data/cc-proxy.conf"
+upstart_destr_file="/etc/init/cc-proxy.conf"
+
+
+# If we are running under the metrics CI system then we do not want the proxy
+# to be dynmaically changing the KSM settings under us - we need control of them
+# ourselves
+if [[ $METRICS_CI ]]; then
+	proxy_systemd_file=$(sudo systemctl show cc-proxy | fgrep FragmentPath | awk 'BEGIN{FS="="}{print $"}')
```

Comment:
The `print $"` doesn't look quite right? (only a single double-quote and no variable?)

Reply:
yep, typo - s/"/2/ (slip of the shift key). thx